template/Makefile.j2

```diff
@@ -0,0 +1,50 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DON'T MANUALLY EDIT THIS FILE
+# =============
+
+.PHONY: docker chart-lint compile-chart
+
+TAG    := $(shell git rev-parse --short HEAD)
+
+VERSION := $(cargo metadata --format-version 1 | jq '.packages[] | select(.name=="stackable-{[ operator.product_string }]-operator") | .version')
+
+docker:
+	docker build --force-rm -t "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:${TAG}" -t "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:latest" -f docker/Dockerfile .
+	echo "${NEXUS_PASSWORD}" | docker login --username github --password-stdin docker.stackable.tech
+	docker push --all-tags docker.stackable.tech/stackable/{[ operator.product_string }]-operator
+
+## Chart related targets
+compile-chart: version crds config
+
+chart-clean:
+	rm -rf deploy/helm/{[ operator.product_string }]-operator/configs
+	rm -rf deploy/helm/{[ operator.product_string }]-operator/templates/crds.yaml
+
+version:
+	sed -i deploy/helm/{[ operator.product_string }]-operator/Chart.yaml "<version_placeholder>" ${VERSION}
+
+config: deploy/helm/{[ operator.product_string }]-operator/configs
+
+deploy/helm/{[ operator.product_string }]-operator/configs:
+	cp -r deploy/config-spec deploy/helm/{[ operator.product_string }]-operator/configs
+
+crds: deploy/helm/{[ operator.product_string }]-operator/templates/crds.yaml
+
+deploy/helm/{[ operator.product_string }]-operator/templates/crds.yaml:
+	cat deploy/crd/*.yaml | yq e '.metadata.annotations["helm.sh/resource-policy"]="keep"' - > ${@}
+
+chart-lint: compile-chart
+	docker run -it -v $(shell pwd):/build/helm-charts -w /build/helm-charts quay.io/helmpack/chart-testing:v3.4.0  ct lint --config deploy/helm/ct.yaml
+
+kind:
+	kind create cluster --name chart-test || exit 0
+	kubectl --context kind-chart-test config view --minify --flatten > /tmp/kind.yaml
+	sleep 10
+	kubectl --kubeconfig /tmp/kind.yaml config set-cluster kind-chart-test --server=https://$(shell docker inspect chart-test-control-plane | jq '.[].NetworkSettings.Networks.kind.IPAddress' -r):6443
```

Comment:
Not sure what you're trying to do here.. isn't this generally covered by `kind export kubeconfig --kubeconfig /tmp/kind.yaml`?